app/assets/javascripts/blacklight/blacklight.js

```diff
@@ -229,29 +229,29 @@ Blacklight.onLoad(function () {
 /*global Blacklight */
 
 
-(function ($) {
-  'use strict';
+'use strict';
```

Comment:
Note: this code is the compiled distribution. The source is in `app/javascript/`